Mathlib/NumberTheory/NumberField/Basic.lean

```diff
@@ -393,6 +393,11 @@ instance numberField : NumberField ℚ where
 noncomputable def ringOfIntegersEquiv : 𝓞 ℚ ≃+* ℤ :=
   RingOfIntegers.equiv ℤ
 
+theorem coe_ringOfIntegersEquiv (z : 𝓞 ℚ) :
```

Comment:
Do you want this to be simp?
```suggestion
@[simp]
theorem coe_ringOfIntegersEquiv (z : 𝓞 ℚ) :
```

Reply:
Sure why not, let's get this def with no API simped out of the way whenever possible. I've looked at the other places where `ringOfIntegersEquiv` is used and it doesn't seem to help but it doesn't seem to hinder either.